app/components/app/app.component.ts

```diff
@@ -13,12 +13,25 @@ import {LoginComponent} from "../login/login.component";
 import {BookingOverviewComponent} from '../bookings/overview/booking-overview.component';
 import {WorkpackageOverviewComponent} from '../workpackages/overview/workpackage-overview.component';
 
+import {Http} from '@angular/http';
+import {TranslateService, TranslatePipe, TranslateLoader, TranslateStaticLoader,} from 'ng2-translate/ng2-translate';
+
 /**
  * Entry point for the App. Holds the Menu and handles which View should be rendered
  */
 @Component({
-  templateUrl: 'build/components/app/app.component.html'
+  templateUrl: 'build/components/app/app.component.html',
+  pipes: [TranslatePipe],
+  providers: [
+    {
+      provide: TranslateLoader,
+      useFactory: (http: Http) => new TranslateStaticLoader(http, 'assets', '.json'),
+      deps: [Http]
+    },
```

Comment:
@mamahler: to have a centralized place to bootstrap the App and maintain providers and Services. Also that's the way we have been doing it for now, I see no reason to make an exception here.